### ISSUE TITLE
Why SGD optimizer?

### ISSUE THREAD

**Issue by Rajat-Mehta** [layumi/Person_reID_baseline_pytorch#174]
Hi,

I wanted to know is there any specific reason that you are using SGD with momentum optimizer instead of more recent variants like Adam and AdaGrad?

How will the model perform if I use Adam? Would you suggest doing so?

Thanks

**Comment by layumi** [comment]
Hi @Rajat-Mehta ,
I follow the traditional ResNet training method in the [original paper](http://openaccess.thecvf.com/content_cvpr_2016/papers/He_Deep_Residual_Learning_CVPR_2016_paper.pdf).

You also could try Adam, and carefully set the learning rate (for example, 3e-4). It would work.
